src/window/empire.c

```diff
@@ -488,60 +542,42 @@ static void handle_input(const mouse *m, const hotkeys *h)
     }
 }
 
-static int is_mouse_hit(tooltip_context *c, int x, int y, int size)
+static int is_mouse_hit(tooltip_context* c, int x, int y, int size)
 {
     int mx = c->mouse_x;
     int my = c->mouse_y;
     return x <= mx && mx < x + size && y <= my && my < y + size;
 }
 
-static int get_tooltip_resource(tooltip_context *c)
+static int get_tooltip_resource(tooltip_context* c)
```

Comment:
```suggestion
static int get_tooltip_resource(tooltip_context *c)
```

Move the `*` to the right place, please.